### ISSUE TITLE
Restarting training fails with Out of range error on PaddingFIFOQueue

### ISSUE THREAD

**Issue by apcode** [google/seq2seq#223]
I trained a model for 100k training steps. I then tried to restart training for another 100k steps, restoring from a saved checkpoint. However it fails to start training with what looks like empty training queues.

Any ideas?
----
INFO:tensorflow:Restoring parameters from model_words/model.ckpt-100000
2017-05-14 16:07:37.639994: W tensorflow/core/framework/op_kernel.cc:1152] Out of range: Reached limit of 1
         [[Node: dev_input_fn/parallel_read/filenames/limit_epochs/CountUpTo = CountUpTo[T=DT_INT64, _class=["loc:@dev_input_fn/parallel_read/filenames/limit_epochs/epochs"], limit=1, _device="/job:localhost/replica:0/task:0/cpu:0"](dev_input_fn/parallel_read/filenames/limit_epochs/epochs)]]
2017-05-14 16:12:44.103782: W tensorflow/core/framework/op_kernel.cc:1152] Out of range: PaddingFIFOQueue '_7_dev_input_fn/batch_queue/padding_fifo_queue' is closed and has insufficient elements (requested 64, current size 0)
         [[Node: dev_input_fn/batch_queue = QueueDequeueUpToV2[component_types=[DT_INT32, DT_STRING, DT_INT32, DT_STRING], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](dev_input_fn/batch_queue/padding_fifo_queue, dev_input_fn/batch_queue/n)]]
2017-05-14 16:12:44.103944: W tensorflow/core/framework/op_kernel.cc:1152] Out of range: PaddingFIFOQueue '_7_dev_input_fn/batch_queue/padding_fifo_queue' is closed and has insufficient elements (requested 64, current size 0)
         [[Node: dev_input_fn/batch_queue = QueueDequeueUpToV2[component_types=[DT_INT32, DT_STRING, DT_INT32, DT_STRING], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](dev_input_fn/batch_queue/padding_fifo_queue, dev_input_fn/batch_queue/n)]]
2017-05-14 16:12:44.104006: W tensorflow/core/framework/op_kernel.cc:1152] Out of range: PaddingFIFOQueue '_7_dev_input_fn/batch_queue/padding_fifo_queue' is closed and has insufficient elements (requested 64, current size 0)
         [[Node: dev_input_fn/batch_queue = QueueDequeueUpToV2[component_types=[DT_INT32, DT_STRING, DT_INT32, DT_STRING], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](dev_input_fn/batch_queue/padding_fifo_queue, dev_input_fn/batch_queue/n)]]
INFO:tensorflow:Finished evaluation at 2017-05-14-15:12:44
INFO:tensorflow:Saving dict for global step 100000: global_step = 100000, log_perplexity = 3.83739, loss = 3.83706
WARNING:tensorflow:Skipping summary for global_step, must be a float or np.float32.

**Comment by sheerun** [comment]
similar issue for me
